packages/plugins/rubygem-smart_proxy_remote_execution_ssh/rubygem-smart_proxy_remote_execution_ssh.spec

```diff
@@ -7,18 +7,24 @@
 
 Summary: SSH remote execution provider for Foreman smart proxy
 Name: rubygem-%{gem_name}
-Version: 0.2.0
+Version: 0.2.1
```

Comment:
Can you reset the release to 1? Thats the test that is failing.

Reply:
aye aye captain. updated